model/gurps/skill.go

```diff
@@ -524,10 +524,11 @@ func (s *Skill) DefaultSkill() *Skill {
 	if e == nil {
 		return nil
 	}
+	requirePoints := e.SheetSettings.UseSkillTrees
```

Comment:
Given the default should be false for UseSkillTrees, this (and the next change) is essentially inverting the current logic, which shouldn't be the case.